base_location_geonames_import/tests/test_base_location_geonames_import.py

```diff
@@ -166,3 +176,93 @@ def test_import_duplicated_city_name(self):
                 state,
                 "Incorrect state for {} {}".format(state_name, zip_code),
             )
+
+    def test_import_countries(self):
+        max_import = 10
+        self.wizard_2.with_context(max_import=max_import).run_import()
+        # Look if there are imported states for the country
+        state_country_2_count = self.env["res.country.state"].search_count(
+            [("country_id", "=", self.country_2.id)]
+        )
+        state_country_3_count = self.env["res.country.state"].search_count(
+            [("country_id", "=", self.country_3.id)]
+        )
+        self.assertTrue(state_country_2_count)
```

Comment:
Why these asserts? You have to test features of the module, not other things. Here you are asserting the previous state of the test DB, which is not desirable. And focus the test to make sure that cities in multiple imported countries are imported, not in other things.

Reply:
Sorry, I'm not sure if I've understood what you mean. country_2 and country_3 are only used at this test, which follows the steps followed by test_import_country. Should I do the test with more than two countries and focus just on the imported cities (and not the states and zips related to each country)? Thank you.

Reply:
OK, I see, I got confused with so many counts. No need to test each object, as checking that there are zips, it's enough to test that the import has worked in this case. Also put `limit=1` for better performance.